### PR TITLE
Minor adjustments to help with Powershell versions

### LINKED ISSUE #83: Powershell Script isn't digitially signed,
### Environment

How do you use Sentry?
Sentry.io

Which version of the SDK?
Latest 

How did you install the package? (Git-URL, Assetstore)
Downloaded from Github manually (And then manually integrated the dependencies from the release pipeline)

Which version of Unreal?
Unreal 5.0.0 

Is this happening in Unreal (editor) or on a player like Android, iOS, Windows?
Editor. 

### Steps to Reproduce

1. Compile the editor 

### Expected Result
The script works out of the box. 

### Actual Result

** For PROJECTEditor-Win64-Development **
[1/1] Executing post-build script (PostBuild-1.bat) (0:00.42 at +0:00)
File C:\Workspaces\PROJECT\PROJECT\Plugins\External\SentryUnrealPlugin\Scripts\upload-debug-symbols-win.ps1
cannot be loaded. The file
C:\Workspaces\PROJECT\PROJECT\Plugins\External\SentryUnrealPlugin\Scripts\upload-debug-symbols-win.ps1 is not
digitally signed. You cannot run this script on the current system. For more information about running scripts and
setting execution policy, see about_Execution_Policies at https:/go.microsoft.com/fwlink/?LinkID=135170.
    + CategoryInfo          : SecurityError: (:) [], ParentContainsErrorRecordException
    + FullyQualifiedErrorId : UnauthorizedAccess

Error executing C:\Windows\system32\cmd.exe (tool returned code: 1)

### PR DESCRIPTION
**Pull request by Edstub207**
Closes #83

### CODE REVIEW

File: plugin-dev/Sentry.uplugin [hunk 1]
```diff
@@ -31,7 +31,7 @@
 			"sh $(PluginDir)/Scripts/upload-debug-symbols.sh $(TargetPlatform) $(TargetName) $(TargetType) $(ProjectDir) $(PluginDir)"
 		],
 		"Win64": [
-			"powershell -ExecutionPolicy RemoteSigned -File $(PluginDir)/Scripts/upload-debug-symbols-win.ps1 $(TargetPlatform) $(TargetName) $(TargetType) $(ProjectDir) $(PluginDir)"
+			"pwsh -ExecutionPolicy bypass -File $(PluginDir)/Scripts/upload-debug-symbols-win.ps1 $(TargetPlatform) $(TargetName) $(TargetType) $(ProjectDir) $(PluginDir)"
```
Comment (vaind) [inline plugin-dev/Sentry.uplugin:34]: Can we stick to `powershell` which should be installed by default?

same above in the `ps1`

Comment (Edstub207) [reply plugin-dev/Sentry.uplugin:34]: It wasn't running on the base version of PowerShell, I can't find the thread right now, but @tustanivsky had similar problems with the default powershell version.

Comment (tustanivsky) [reply plugin-dev/Sentry.uplugin:34]: I recall having issues with setting `SENTRY_PROPERTIES` env variable when using the default PowerShell version, though I wasn't able to reproduce it lately.

Comment (vaind) [reply plugin-dev/Sentry.uplugin]: I remember that, but there have been changes in the meantime. Let's see if that helps (in the next release) and get back to this if it doesn't.

**Comment by vaind** [comment]
Seems to be fine as is, let's stick to the preinstalled powershell for now and get back to this when an issue arises.

**Pull request closed without merging**